src/main/java/net/mcreator/ui/modgui/LivingEntityGUI.java

```diff
@@ -454,6 +464,24 @@ private synchronized void regenerateAITasks() {
 				L10N.label("elementgui.living_entity.water_mob")));
 		subpane1.add(waterMob);
 
+		waterMob.addChangeListener(e -> {
+			if (!isEditingMode() && waterMob.isSelected()) {

```

Comment:
This can be simplified so that fixed values are set at the same `if` branch. I think conditions could look a bit clearer in this case.